src/model/jwks.rs

```diff
@@ -119,7 +126,14 @@ impl Jwk {
 
         let modulus: Vec<u8> = rsa.n().to_vec();
         let exponent: Vec<u8> = rsa.e().to_vec();
-        let pkey: PKey<Private> = PKey::from_rsa(rsa)?;
+        let key_pair: PKey<Private> = PKey::from_rsa(rsa)?;
+
+        let x509 = generate_x509_cert(&key_pair)?;
+        let x509pem = x509.to_pem()?;
+        let x509cert = String::from_utf8(x509pem)?
+            .replace('\n', "")
+            .replace("-----BEGIN CERTIFICATE-----", "")
+            .replace("-----END CERTIFICATE-----", "");
```

Comment:
I trust that there is no better way to do this in the openssl API but for the record I'm not a fan of doing it this way

Reply:
We didn't find a openssl method to do this operation, the official auth0 library for PHP does the [same thing](https://github.com/auth0/auth0-PHP/blob/3242a82ba3577254752719d68b863cf2f18efc3c/src/Token/Verifier.php#L258).